components/src/core/Hero/Hero.tsx

```diff
@@ -117,20 +120,24 @@ const HeroRender: React.ForwardRefRenderFunction<unknown, HeroProps> = (props: H
             ref={ref}
             className={cx(defaultClasses.root, classes.root, userClassName)}
             data-test={'wrapper'}
-            iconSize={iconSize}
-            iconBackgroundColor={iconBackgroundColor}
             {...otherProps}
         >
-            <span className={cx(defaultClasses.icon, classes.icon)}>{icon}</span>
-            <span className={cx(defaultClasses.values, classes.values)}>
+            <Icon
+                className={cx(defaultClasses.icon, classes.icon)}
+                iconSize={iconSize}
+                iconBackgroundColor={iconBackgroundColor}
+            >
+                {icon}
+            </Icon>
+            <Values className={cx(defaultClasses.values, classes.values)}>
```

Comment:
The `Box` component is a `div` by default. Label and Values should be `span`.

```suggestion
            <Values  component={'span'} className={cx(defaultClasses.values, classes.values)}>
```